config/history.go

```diff
@@ -61,7 +87,11 @@ var DefaultChains = map[string]*History{
 				// https://github.com/oasisprotocol/mainnet-artifacts/releases/tag/2022-04-11
 				ArchiveName:   "damask",
 				GenesisHeight: 8048956,
-				ChainContext:  "b11b369e0da5bb230b220127f5e7b242d385ef8c6f54906243f30af63c815535",
+				RuntimeStartRounds: map[common.Runtime]uint64{
+					common.RuntimeCipher:  8284,
```

Comment:
Optional: Let's add a comment about where/how these numbers can be obtained. When the next archive comes around (to say nothing of the one after next), the person updating this will be thankful.

Reply:
adding in RuntimeStartRounds doc comment

Reply:
wait no, already there

Reply:
> 	// RuntimeStartRounds has entries for runtimes that already exist at the
	// genesis of this network. Look these up in the genesis document's
	// .roothash.runtime_states[runtime_id_hex].round.